doc/conf.py

```diff
@@ -292,7 +292,18 @@
         'sklearn': 'http://scikit-learn.org/stable'}
     }
 
+
+def touch_example_backreferences(app, what, name, obj, options, lines):
```

Comment:
Not a blocker to merge this PR, but should this be in sphinx_gallery itself? It seems that it's something that people can easily get wrong.